bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -480,15 +484,94 @@ esp_err_t bsp_display_new_with_handles(const bsp_display_config_t *config, bsp_l
+
+#elif CONFIG_BSP_LCD_TYPE_HDMI
+
+#if !CONFIG_BSP_LCD_COLOR_FORMAT_RGB888
+#error The color format must be RGB888 in HDMI display type!
+#endif
+    ESP_LOGI(TAG, "Install MIPI DSI HDMI control panel");
+    ESP_RETURN_ON_ERROR(bsp_i2c_init(), TAG, "I2C init failed");
+
+    /* Main IO */
+    esp_lcd_panel_io_i2c_config_t io_config = LT8912B_IO_CFG(CONFIG_BSP_I2C_CLK_SPEED_HZ, LT8912B_IO_I2C_MAIN_ADDRESS);
+    ESP_ERROR_CHECK(esp_lcd_new_panel_io_i2c(i2c_handle, &io_config, &io));
+
+    /* CEC DSI IO */
+    esp_lcd_panel_io_handle_t io_cec_dsi = NULL;
+    esp_lcd_panel_io_i2c_config_t io_config_cec = LT8912B_IO_CFG(CONFIG_BSP_I2C_CLK_SPEED_HZ, LT8912B_IO_I2C_CEC_ADDRESS);
+    ESP_ERROR_CHECK(esp_lcd_new_panel_io_i2c(i2c_handle, &io_config_cec, &io_cec_dsi));
+
+    /* AVI IO */
+    esp_lcd_panel_io_handle_t io_avi = NULL;
+    esp_lcd_panel_io_i2c_config_t io_config_avi = LT8912B_IO_CFG(CONFIG_BSP_I2C_CLK_SPEED_HZ, LT8912B_IO_I2C_AVI_ADDRESS);
+    ESP_ERROR_CHECK(esp_lcd_new_panel_io_i2c(i2c_handle, &io_config_avi, &io_avi));
+
+    /* DPI config */
+#if CONFIG_BSP_LCD_HDMI_800x600_60HZ
+    ESP_LOGI(TAG, "HDMI configuration for 800x600@60HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_800x600_PANEL_60HZ_DPI_CONFIG();
+#elif CONFIG_BSP_LCD_HDMI_1024x768_60HZ
+    ESP_LOGI(TAG, "HDMI configuration for 1024x768@60HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_1024x768_PANEL_60HZ_DPI_CONFIG();
+#elif CONFIG_BSP_LCD_HDMI_1280x720_60HZ
+    ESP_LOGI(TAG, "HDMI configuration for 1280x720@60HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_1280x720_PANEL_60HZ_DPI_CONFIG();
+#elif CONFIG_BSP_LCD_HDMI_1280x800_60HZ
+    ESP_LOGI(TAG, "HDMI configuration for 1280x800@60HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_1280x800_PANEL_60HZ_DPI_CONFIG();
+#elif CONFIG_BSP_LCD_HDMI_1920x1080_30HZ
+    ESP_LOGI(TAG, "HDMI configuration for 1280x800@30HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_1920x1080_PANEL_30HZ_DPI_CONFIG();
+#else
+    ESP_LOGI(TAG, "HDMI configuration for 1920x1080@60HZ");
+    esp_lcd_dpi_panel_config_t dpi_config = LT8912B_1920x1080_PANEL_60HZ_DPI_CONFIG();
 #endif
+    dpi_config.num_fbs = CONFIG_BSP_LCD_DPI_BUFFER_NUMS;
+    lt8912b_vendor_config_t vendor_config = {
+#if CONFIG_BSP_LCD_HDMI_800x600_60HZ
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_800x600_60Hz(),
+#elif CONFIG_BSP_LCD_HDMI_1024x768_60HZ
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_1024x768_60Hz(),
+#elif CONFIG_BSP_LCD_HDMI_1280x720_60HZ
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_1280x720_60Hz(),
+#elif CONFIG_BSP_LCD_HDMI_1280x800_60HZ
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_1280x800_60Hz(),
+#elif CONFIG_BSP_LCD_HDMI_1920x1080_30HZ
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_1920x1080_30Hz(),
+#else
+        .video_timing = ESP_LCD_LT8912B_VIDEO_TIMING_1920x1080_60Hz(),
```

Comment:
You have removed the related Kconfig option but this code is still here...

Reply:
I am keeping this in code for feature testing and continue looking for solution for it. Only removed from readme and Kconfig for hide for customers.

Reply:
Okay, that works for me, was just wondering. Maybe leaving a comment that this config doesn't work for the time being would be nice, so that users who notice it in the code don't waste time trying to get it to work.

Reply:
ok, updated.